.github/workflows/test-suite-desktop-e2e.yml

```
@@ -96,15 +116,15 @@ jobs:
           yarn workspace @trezor/suite-desktop-core test:e2e:desktop --forbid-only --grep=${{ matrix.TEST_GROUP }}
 
       - name: Extract Trezor-user-env and Regtest logs
-        if: ${{ ! cancelled() }}
+        if: ${{ ! cancelled() }} && steps.check-previous-runs.outputs.skip_tests != 'true'
```

actionlint (1.7.4)

if: condition "${{ ! cancelled() }} && steps.check-previous-runs.outputs.skip_tests != 'true'" is always evaluated to true because extra characters are around ${{ }}

(if-cond)